profileservice/profile.yaml

```diff
@@ -190,6 +190,47 @@ paths:
           description: Profile not found
         500:
           description: Internal error
+  /profiles/core:
+    get:
+      tags:
+      - PII
+      summary: Get profile data by query params
+      description: |
+        Returns a single PII entry with associated Non PII information.
+
+        Auth: Requires a valid API Key for access. The API Key must match the value of environment variable CORE_BB_API_KEY.
```

Comment:
I suppose that this API key is used only in the backend. Please clarify if otherwise. Thanks.